src/components/templates/ConversationSpace/ConversationSpace.tsx

```diff
@@ -41,118 +23,40 @@ export interface ConversationSpaceProps {
 export const ConversationSpace: React.FC<ConversationSpaceProps> = ({
   venue,
 }) => {
-  const analytics = useAnalytics({ venue });
-
-  const { parentVenue, parentVenueId } = useRelatedVenues({
-    currentVenueId: venue?.id,
-  });
-
-  const {
-    isShown: showOnlyAvailableTables,
-    toggle: toggleTablesVisibility,
-  } = useShowHide();
-
-  const [seatedAtTable, setSeatedAtTable] = useState<string>();
-
-  useUpdateTableRecentSeatedUsers(
-    VenueTemplate.conversationspace,
-    seatedAtTable && venue?.id
-  );
+  useBackgroundGradient();
 
-  useEffect(() => {
-    if (seatedAtTable) analytics.trackSelectTableEvent();
-  }, [analytics, seatedAtTable]);
+  const { userId } = useUser();
```

Comment:
Deleting code looks good and I think I'd have gone with few more adjustments:
- rename `venue` to `space`
- `useUserId()` instead of `useUser()` because it doesn't bother reading extra profile data from DB
- make a `ConversationSpaceHoc` wrapper component for the data

```tsx
  if (!userId) {
    return <Loading />;
  }

  return <ConversationSpace userId={userId} space={space} />
```

and just add this plumbing into `index.ts`:
```tsx
export { ConversationSpaceHoc as ConversationSpace } from "./ConversationSpaceHoc";
```

Reply:
a separate hoc: https://github.com/sparkletown/sparkle/pull/2910/commits/afe06cdd59c668832f14435166c1d3c97fba4064 
use useUserId hook: https://github.com/sparkletown/sparkle/pull/2910/commits/cbe7671c39d0f0d7fea3653309aef9b13aa2d5d6